sktime/utils/strtobool.py

```diff
@@ -0,0 +1,35 @@
+"""String-to-int coercion utility strtobool formerly in deprecated distutils."""
+
+
+# distutils has been removed in Python 3.12,
+# but this function is still used in parts of the sktime codebase
+def strtobool(val):
```

Comment:
Another non-blocking: the module and function names are a bit misleading, it's not returning booleans.

(And I just noticed that you also said so in the module docstring as well.)

Reply:
that is also directly copied from `distutils` - both the name and the function (which had no docstring in `distutils`)